contrib/clojure-package/ci-test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+
```

Comment:
It looks like this file head trouble running in CI 
```+ ./contrib/clojure-package/ci-test.sh
/work/runtime_functions.sh: line 659: ./contrib/clojure-package/ci-test.sh: Permission denied
```
change file permissions to 755?

Reply:
+1

Reply:
Will address this and other changes later tonight. Thanks for your feedback.